### PR TITLE
Add simulator tracing, and debug port configuration via environment v…

### PR DESCRIPTION
**Pull request by ronlieb**
Adds environment variable control to enable simulator tracing, either full or minimal.

also adds debugger port configuration, which only woks if we call Run() API in the simulator. which at present we do not.

### CODE REVIEW

File: src/runtime/hexagon_remote/sim_host.cpp [hunk 1]
```diff
@@ -29,6 +29,39 @@ int init_sim() {
         return -1;
     }
 
+    // Configue tracing.
+    const char *T = getenv("HALIDE_HEX_SIM_MIN_TRACE");
+    if (T && T[0] != 0) {
+        status - sim->SetTracing(HEX_TRACE_PC_MIN, T);
+        if (status != HEX_STAT_SUCCESS) {
+            fprintf(stderr,"HexagonWrapper::SetTracing MIN failed: %d\n", statu
```
Comment (dsharletg) [inline src/runtime/hexagon_remote/sim_host.cpp]: Can this just use printf? Remember this isn't simulator side code running, so the buffering issues shouldn't be a problem. It's what the rest of the code in this file does.

File: src/runtime/hexagon_remote/sim_host.cpp [hunk 2]
```diff
@@ -29,6 +29,39 @@ int init_sim() {
...
+            return -1;
+        }
+    } else {
+        const char *T = getenv("HALIDE_HEX_SIM_TRACE");
+        if (T && T[0] != 0) {
+             status - sim->SetTracing(HEX_TRACE_PC, T);
+            if (status != HEX_STAT_SUCCESS) {
+               fprintf(stderr,"HexagonWrapper::SetTracing failed: %d\n", status
```
Comment (dsharletg) [inline src/runtime/hexagon_remote/sim_host.cpp]: It looks like there is a missing paren at the end of this?

File: src/runtime/hexagon_remote/sim_host.cpp (hunk 1 above)
Comment (dsharletg) [inline src/runtime/hexagon_remote/sim_host.cpp]: Also, missing paren at the end?

**Comment by ronlieb** [comment]
having trouble with my local repo, closing, pushinga  new one

**Pull request closed without merging**